### ISSUE TITLE
[Feature request] Adjust wallet balance in REST API

### ISSUE THREAD

**Issue by kieusonlam** [malsubrata/woo-wallet#30]
Currently, we can "Insert new transaction", it would be greate if we can modify the balance as we wish.

**Comment by AffiliateMarket** [comment]
I agree. Having the ability for admin to control these things would be very important in so many circumstances. Including for the reason I've made a few "accidents" and now I can't do anything about it now.
I even tried to delete the plugin and reinstall... It ruined my website.
If you are to have a wallet plugin on your site, you definitely want full control. You can't have an "add" option and not have a "remove" option. Especially if you find your users frauding the wallet system and you can't "remove" anything.

Very important to add this feature. Or maybe someone on fiverr will make this.
